common/config/rush/pnpm-lock.yaml

```diff
@@ -1049,6 +1049,19 @@ packages:
       tslib: 2.3.1
     dev: false
 
+  /@azure/mixed-reality-authentication/1.0.0-beta.1:
```

Comment:
There're many changes here that I do not expect.

Could you reset this file by doing:
`git fetch --all && git checkout <upstream/main> common/config/rush/pnpm-lock.yaml` where `<upstream/main>` is the remote main branch in this repository.

and then run `rush update`?